modules/backend/formwidgets/DatePicker.php

```diff
@@ -89,7 +96,11 @@ public function prepareVars()
 
 
         if ($value = $this->getLoadValue()) {
-            $value = $value instanceof Carbon ? $value->toDateTimeString() : $value;
+            if ($value instanceof Carbon) {
+                $value = $value->toDateTimeString();
+            } elseif ($value == $this->default) {
```

Comment:
It would appear `$this->default` is not even a valid property, it will always return null...